lib/omsmailer_web/templates/page/candidate_applied.html.eex

```diff
@@ -0,0 +1,82 @@
+
+<body style="width:100% !important;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;margin-top:0;margin-bottom:0;margin-right:0;margin-left:0;padding-top:0;padding-bottom:0;padding-right:0;padding-left:0;">
+  <table cellpadding="0" cellspacing="0" border="0" id="backgroundTable" style="margin:0; padding:0; width:100% !important; line-height: 100% !important; border-collapse:collapse; mso-table-lspace:0pt; mso-table-rspace:0pt;background-color: #F9FAFB;"
+    width="100%">
+    <tbody>
+      <tr>
+        <td width="10" valign="top">&nbsp;</td>
+        <td valign="top" align="center">
+          <table cellpadding="0" cellspacing="0" border="0" align="center" style="width: 100%; max-width: 600px; background-color: #FFF; border-collapse:collapse;mso-table-lspace:0pt;mso-table-rspace:0pt;"
+            id="contentTable">
+            <tbody>
+              <tr>
+                <td width="600" valign="top" align="center" style="border-collapse:collapse;">
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="background: #F9FAFB;" width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;">&nbsp;</div>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="border: 1px solid #E0E4E8;"
+                    width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" style="padding: 56px 56px 28px 56px;" valign="top">
+                          <a style="color: #3498DB; text-decoration: none;" href="my.aegee.eu"
+                            target="_blank"><img style="width: 56px; height: 56px; border: 0;" alt="Logo"
+                              src="https://my.aegee.eu/images/logo.png">
+                          </a></td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 20px; color: #333;"><strong>A new application is submitted</strong></div>
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 16px; color: #333;">
+                            A new application to position
+                            <strong><%= @parameters["position"]["name"] %></strong>
+                            is submitted by
+                            <strong><%= @parameters["candidate"]["first_name"] @parameters["candidate"]["last_name"] %></strong>.
```

Comment:
Can you add multiple parameters together like this? In another email we're doing these seperately

Reply:
I don't know, since Elixir is a mystery, but I can split them to make sure

Reply:
don't think it'll work, try this instead

```
<strong><%= @parameters["candidate"]["first_name"]%> <%=@parameters["candidate"]["last_name"] %></strong>. 
```